aten/src/ATen/native/cuda/MultiLabelMarginCriterion.cu

```diff
@@ -311,19 +308,21 @@ void multilabel_margin_loss_backward_cuda_out_template(
     int64_t reduction,
     const Tensor& is_target,
     Tensor& grad_input) {
-  check_shape(input, target);
-  auto input_ = input.contiguous();
-  if (input_.numel() == 0) {
+  int64_t nframe, dim;
+  const int64_t ndims = input.dim();
+  multilabel_margin_loss_shape_check(nframe, dim, ndims, input, target);
+
+  if (input.numel() == 0) {
     return;
   }
 
-  grad_input.resize_as_(input_);
+  auto input_ = input.contiguous();
   auto target_ = target.contiguous();
   auto is_target_ = is_target.contiguous();
   auto grad_output_ = grad_output.contiguous();
+  grad_input.resize_as_(input_);
```

Comment:
Moved these statements around to match the forward code, so that all `contiguous` calls are in one place.